mongodbatlas/organizations.go

```diff
@@ -188,3 +212,138 @@ func (s *OrganizationsServiceOp) Delete(ctx context.Context, orgID string) (*Res
 
 	return resp, err
 }
+
+// ListUnacceptedInvitations gets all unaccepted invitations to the specified Atlas organization.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/organization-get-invitations/
+func (s *OrganizationsServiceOp) ListUnacceptedInvitations(ctx context.Context, orgID string, opts *InvitationOptions) (*[]Invitation, *Response, error) {
+	if orgID == "" {
+		return nil, nil, NewArgError("orgID", "must be set")
+	}
+
+	basePath := fmt.Sprintf(invitationBasePath, orgID)
+	path, err := setListOptions(basePath, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.Client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new([]Invitation)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return root, resp, nil
+}
+
+// GetUnacceptedInvitation gets details for one unaccepted invitation to the specified Atlas organization.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/organization-get-one-invitation/
+func (s *OrganizationsServiceOp) GetUnacceptedInvitation(ctx context.Context, orgID, invitationID string) (*Invitation, *Response, error) {
+	if orgID == "" {
+		return nil, nil, NewArgError("orgID", "must be set")
+	}
+
+	if invitationID == "" {
+		return nil, nil, NewArgError("invitationID", "must be set")
+	}
+
+	basePath := fmt.Sprintf(invitationBasePath, orgID)
+	path := fmt.Sprintf("%s/%s", basePath, invitationID)
+
+	req, err := s.Client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new(Invitation)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return root, resp, nil
+}
+
+// InviteUser invites one user to the Atlas organization that you specify.
+//
+// See more: https://docs-atlas-staging.mongodb.com/cloud-docs/docsworker-xlarge/DOCSP-14695/reference/api/organization-create-one-invitation/
```

Comment:
[q] why does it have staging url? I don't think lib users have access to it. Better remove it or add a TODO to update it later.

Reply:
my mistakes, I've used all the links in the mongocli scope which are of staging